crates/ruma-events/src/sticker.rs

```diff
@@ -4,9 +4,67 @@
 
 use ruma_common::OwnedMxcUri;
 use ruma_macros::EventContent;
-use serde::{Deserialize, Serialize};
+use serde::{de, Deserialize, Serialize};
 
-use crate::room::ImageInfo;
+#[cfg(feature = "compat-encrypted-stickers")]
+use crate::room::EncryptedFile;
+use crate::room::{ImageInfo, MediaSource};
+
+/// The source of a sticker media file.
+#[derive(Clone, Debug, Serialize)]
+#[allow(clippy::exhaustive_enums)]
```

Comment:
This enum needs to be non-exhaustive:

```suggestion
#[non_exhaustive]
```